horace_core/Tobyfit/Tobyfit/tobyfit_refine_moderator_options.m

```diff
@@ -5,7 +5,9 @@
 % Please migrate to the new Tobyfit syntax supported since January 2018'
 % For details of the new syntax:  <a href="matlab:doc(''sqw/tobyfit'');">Click here</a>'
 
-mess = ['\nThis function was used with the obsolete Tobyfit syntax.',...
-    '\nPlease migrate to the new Tobyfit syntax supported since January 2018',...
-    '\nFor details of the new syntax:  <a href="matlab:doc(''sqw/tobyfit'');">Click here</a>'];
-error('Tobyfit:legacySyntax',mess)
+error('HORACE:tobyfit:deprecated', ...
```

Comment:
all these errors have identical output. Would it make sense to specify which call triggered it?

Reply:
That'll appear in the backtrace